Add information about tlsVerify: false on AKS with new Agent versions (#14197)
DataDog/documentation: content/en/agent/kubernetes/distributions.md

        fieldRef:
          fieldPath: spec.nodeName
    hostCAPath: /etc/kubernetes/certs/kubeletserver.crt

>     tlsVerify: false # Since Agent 7.35, tlsVerify: false is currently required as Kubelet certificates in AKS do not have any SAN (Subject Alternative Name) set

Reviewer: ```suggestion
    tlsVerify: false # Required as of Agent 7.35. See Notes.
```

          fieldRef:
            fieldPath: spec.nodeName
        hostCAPath: /etc/kubernetes/certs/kubeletserver.crt

>         tlsVerify: false # Since Agent 7.35, tlsVerify: false is currently required as Kubelet certificates in AKS do not have any SAN (Subject Alternative Name) set

Reviewer: ```suggestion
        tlsVerify: false # Required as of Agent 7.35. See Notes.
```

{{% /tab %}}
{{< /tabs >}}

**Notes**:

> - Since Agent 7.35, tlsVerify: false is currently required as Kubelet certificates in AKS do not have any SAN (Subject Alternative Name) set.

Reviewer: ```suggestion
- As of Agent 7.35, `tlsVerify: false` is required because Kubelet certificates in AKS do not have a Subject Alternative Name (SAN) set.
```
